core-implementation/get_account_balance/get_account_balance.ts

```diff
@@ -0,0 +1,38 @@
+import { handleUnaryCall } from "@grpc/grpc-js";
+import { core } from "../../generated/core/core.js";
+// Imports the transaction ledger to search
+import { ledger } from "../search_transaction_by_user/search_transaction_by_user.js";
+
+
+// Define handler for getAccountBalance, which takes the user id and returns a CurrencyValue response.
+export const getAccountBalance: handleUnaryCall<
+    core.UserId,
+    core.CurrencyValue
+> = (request, respond) => {
+    // Extract the user ID from the incoming request.
+    const userID = request.request.user_id;
+
+    // Initializes balance to keep track of changes (sending and receiving)
+    let balance : number = 0;
+
+    // Iterates over the ledger and respectively makes changes to balance
+    // * If receiever --> Adds to balance
+    // * If sender    --> Subtract from balance
+    ledger.forEach(transaction => {
```

Comment:
dont worry about this for sprint 1, i just thought you should know:

when we do this for real, we will have to keep track of the balances of each user separately from the transactions to avoid iterating over each transaction to calculate a balance.
for example, you might have an object keyed by the user's id that stores the user's balance.